packages/plugin-ext/src/main/node/plugin-ext-backend-module.ts

```diff
@@ -40,8 +40,10 @@ import { PluginTheiaEnvironment } from '../common/plugin-theia-environment';
 import { PluginTheiaDeployerParticipant } from './plugin-theia-deployer-participant';
 import { WebviewBackendSecurityWarnings } from './webview-backend-security-warnings';
 import { PluginUninstallationManager } from './plugin-uninstallation-manager';
+import { LocalizationBackendContribution } from '@theia/core/src/node/i18n/localization-backend-contribution';
```

Comment:
Minor: Please don't import from `src`.
```suggestion
import { LocalizationBackendContribution } from '@theia/core/lib/node/i18n/localization-backend-contribution';
```